test_phage_reference/nbconverted/1_align_quantify_samples.py

```diff
@@ -0,0 +1,211 @@
+
+# coding: utf-8
+
+# # Aign and quantify samples
+# 
+# This notebook aligns test samples against the phage and PAO1 reference genomes. Our goal is to test our phage reference genome alignment before we port it to the Discovery (Dartmouth computing cluster). We want to check that we are getting more expression of phage genes in a phage sample compared to a non-pseudomonas samples.
+
+# In[17]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+
+import os
+import shutil
+import pandas as pd
+import numpy as np
+from core_acc_modules import paths
+
+np.random.seed(123)
+
+
+# ### Setup SRA toolkit -- only needs to be run once
+
+# In[2]:
+
+
+# Download latest version of compiled binaries of NCBI SRA toolkit 
```

Comment:
Is this only going to be run in notebook form? Because if a future user tries to run these scripts as a plug-and-chug, they may encounter problems with the download info being commented out. But if that scenario's never going to happen don't worry about it.

Reply:
Good point. I have separated out the scripts to setup the analysis from those that run the analysis for future use. But this really is just a quick test so I don't anticipate a user using this. However the notebooks were more of a way for me to organize my thoughts on how this pipeline is working.